source/RSRCFilter.cpp

```diff
@@ -0,0 +1,44 @@
+#include "RSRCFilter.h"
```

Comment:
Please add a copyright header.

Reply:
no email?

Reply:
if you prefer an email, I can change it!